site/_docs/json_reference.md

```diff
@@ -920,6 +934,7 @@ This object represents the properties for a given JDBC Connection.
 {% highlight json %}
 {
   "connProps": "connPropsImpl",
+  "dirty": true,
```

Comment:
This is [`is_dirty`](https://github.com/apache/calcite-avatica/blob/4b7eee5bf430b916c7c07897b6f60d2b6b6dabb7/core/src/main/protobuf/common.proto#L24) for protobuf but seems to be merely `dirty` for JSON.